src/Couchbase.Lite.Shared/Sync/WebSocketWrapper.cs

```diff
@@ -385,31 +375,24 @@ private void OnSocketReady()
         {
             //STEP 5: Send the HTTP request to start the WebSocket upgrade
             var httpData = _logic.HTTPRequestData();
-            var cts = new CancellationTokenSource();
-            cts.CancelAfter(IdleTimeout);
             if (NetworkStream == null) {
                 WriteLog.To.Sync.E(Tag, "Socket reported ready, but no network stream available!");
                 DidClose(C4WebSocketCloseCode.WebSocketCloseAbnormal, "Unexpected error in client logic");
                 return;
             }
 
-            NetworkStream.ReadTimeout = (int)IdleTimeout.TotalMilliseconds;
-            NetworkStream.WriteTimeout = (int)IdleTimeout.TotalMilliseconds;
-            NetworkStream.WriteAsync(httpData, 0, httpData.Length, cts.Token).ContinueWith(t =>
```

Comment:
This write still needs to happen, minus the cancellation.  This is the HTTP connection for the web socket handshake.